dashboards/process_memory/main.yml

```
@@ -0,0 +1,24 @@
+metric_keys:
+  - process_rss
+dashboard:
+  title: Process memory usage
+  description: RSS memory usage of your processes split out per host and process type.
+  graphs:
+    - title: Average memory usage
+      description: Average RSS memory usage of your processes split out per host and process type.
+      line_label: '%hostname% - %name%'
```

all the other line labels we have in the public_config use name first then hostname. We should be consistent here.
```suggestion
      line_label: '%name% - %hostname%'
```